signature/openpgp/openpgp_test.go

```diff
@@ -0,0 +1,102 @@
+package openpgp
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/containers/image/signature"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestImportKeysFromBytes(t *testing.T) {
+	m, _ := NewOpenPGPSigningMechanism()
+	keyBlob, err := ioutil.ReadFile("../fixtures/public-key.gpg")
+	require.NoError(t, err)
+	keyIdentities, err := m.ImportKeysFromBytes(keyBlob)
+	require.NoError(t, err)
+	assert.Equal(t, []string{signature.TestKeyFingerprint}, keyIdentities)
+}
+
+func TestVerify(t *testing.T) {
+
+	type tc struct {
+		fixture      string
+		fixtureBytes []byte
+		isValid      bool
+		identity     string
+		publicKey    string
+	}
+
+	cases := []tc{
```

Comment:
Yes, a table-driven test is nicer, though it would be nice to keep the comments and ideally to use the same set of tests for both implementations.  We can tinker with this after the general structure of the code is fixed, though.